video/out/drm_common.c

```diff
@@ -1047,6 +1167,12 @@ void vo_drm_uninit(struct vo *vo)
     if (!drm)
         return;
 
+    restore_sdr(drm);
```

Comment:
Do we need "restore" function? Wouldn't be better to always call `vo_drm_set_metadata` and act accordingly, set hdr if possible, set sdr otherwise?

Reply:
Well yeah we do have to restore back to sdr on quit otherwise the display will be stuck in hdr mode. We can't rely on the next page flip here either so the atomic commit has also be done in the same step. But probably what it should be is that `restore_sdr` calls `vo_drm_set_hdr_metadata` instead and forces sdr.

Reply:
Refactored the `restore_sdr` and `vo_drm_set_hdr_metadata` logic so there's less redundancy now.